test/utils/testHelpers.js

```diff
@@ -319,18 +316,13 @@ class TestHelper {
 	// Virtual debt = 50 TRI.
```

Comment:
Shouldn't we just get rid of these functions? Or keep them to preserve the least amount of changes?